packages/contentstack-bulk-publish/src/producer/cross-publish.js

```diff
@@ -241,6 +261,49 @@ function setConfig(conf, bp) {
   filePath = initializeLogger(logFileName);
 }
 
+async function getVariantEntries(stack, contentType, entries, index, queryParams, skip = 0) {
+  try {
+    let variantQueryParams = {
+      locale: queryParams.locale || 'en-us',
+      include_count: true,
+      skip: skip, // Adding skip parameter for pagination
+      limit: 100, // Set a limit to fetch up to 100 entries per request
+      include_publish_details: true,
```

Comment:
@sunil-lakshman , include_publish_details not required pls remove it

Reply:
@aman19K Removed.